hat/assets/js/apps/Iaso/domains/app/translations/en.json

```diff
@@ -1098,6 +1098,8 @@
     "iaso.tasks.launcher": "User",
     "iaso.tasks.message": "Message",
     "iaso.tasks.name": "Name",
+    "iaso.tasks.polioNotificationImport.details": "Polio Notifications Import Details",
```

Comment:
Just a question, why in this translation each work begin with a capital letter?

Reply:
Because [title case](https://en.wikipedia.org/wiki/Title_case) is a common rule of most English style guides for rendering the titles.

Should I switch to normal case?

Reply:
I don't know, but I'm wondering if the rule is applied on all title in english in IASO. @quang-le  can may be can help to clarify it?